monitoring/iris/views.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import render
+from datetime import datetime, timedelta
+
+from django.db.models import Max
+from django.db.models.functions import Lower
+
+from rest_framework import viewsets
+from rest_framework.renderers import TemplateHTMLRenderer
+
+
+from monitoring.iris.models import IrisCloudAndGrid
+
+from monitoring.iris.serializers import IrisCloudAndGridSerializer
+
+class IrisViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
The spacing and ordering here isn't very standard, Should be the following, with just one line between earch group.

1st party - `datetime`, 3rd party - `django`, `rest_framework`, app specific - `monitoring`.

And then two blank lines before the class.